src/test/java/teammates/test/pageobjects/AppPage.java

```diff
@@ -812,104 +797,6 @@ private boolean testAndRunGodMode(String filePath, String content, boolean isPar
         }
     }
     
-    private static String processPageSourceForHtmlComparison(String content) {
-        return replaceUnpredictableValuesWithPlaceholders(
-                                        suppressVariationsInInjectedValues(content));
-    }
-
-    private static String suppressVariationsInInjectedValues(String content) {
-        return content // this replaces dev server admin absolute URLs (/_ah/...) with their relative counterparts
-                      .replace("\"" + TestProperties.inst().TEAMMATES_URL + "/_ah", "\"/_ah")
-                      // this replaces all printed version of TEAMMATES tested with the current version
-                      .replaceAll("V[0-9]+(\\.[0-9]+)+", "V" + TestProperties.inst().TEAMMATES_VERSION)
-                      // this replaces truncated long accounts with their original counterpart
-                      .replace(StringHelper.truncateLongId(TestProperties.inst().TEST_STUDENT1_ACCOUNT),
-                               TestProperties.inst().TEST_STUDENT1_ACCOUNT)
-                      .replace(StringHelper.truncateLongId(TestProperties.inst().TEST_STUDENT2_ACCOUNT),
-                               TestProperties.inst().TEST_STUDENT2_ACCOUNT)
-                      .replace(StringHelper.truncateLongId(TestProperties.inst().TEST_INSTRUCTOR_ACCOUNT),
-                               TestProperties.inst().TEST_INSTRUCTOR_ACCOUNT)
-                      .replace(StringHelper.truncateLongId(TestProperties.inst().TEST_ADMIN_ACCOUNT),
-                               TestProperties.inst().TEST_ADMIN_ACCOUNT);
-    }
-
-    private static String replaceUnpredictableValuesWithPlaceholders(String content) {
-        Date now = new Date();
-        assertEquals(new SimpleDateFormat("EEE, dd MMM yyyy, hh:mm a").format(now), TimeHelper.formatTime12H(now));
-        return content
-                // this handles the logout url that google generates
-                .replaceAll("_ah/logout\\?continue=.*?\"", "_ah/logout?continue=\\${continue\\.url}\"")
-                // photo from instructor
-                .replaceAll(Const.ActionURIs.STUDENT_PROFILE_PICTURE + "\\?" + Const.ParamsNames.STUDENT_EMAIL + "=([a-zA-Z0-9]){1,}\\&amp;"
-                        + Const.ParamsNames.COURSE_ID + "=([a-zA-Z0-9]){1,}", 
-                        Const.ActionURIs.STUDENT_PROFILE_PICTURE + "\\?" + Const.ParamsNames.STUDENT_EMAIL 
-                        + "=\\${student\\.email\\.enc}\\&amp;" + Const.ParamsNames.COURSE_ID + "=\\${course\\.id\\.enc}")
-                .replaceAll(Const.ActionURIs.STUDENT_PROFILE_PICTURE + "\\?" + Const.ParamsNames.COURSE_ID + "=([a-zA-Z0-9]){1,}\\&amp;"
-                        + Const.ParamsNames.STUDENT_EMAIL + "=([a-zA-Z0-9]){1,}", 
```

Comment:
@wkurniawan07, I think we had courseid followed by studentemail and studentemail followed by courseid because both can occur depending on the browser being used